internal/lib/rename.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
 
-	"github.com/containers/libpod/pkg/annotations"
+	"github.com/containers/libpod/v2/pkg/annotations"
```

Comment:
can/should the libpod references be changed to podman now?

Reply:
I would not do this now and defer it to a future PR.